src/utils/createAlpineREPL.js

```diff
@@ -0,0 +1,34 @@
+import { getParameters } from 'codesandbox/lib/api/define';
```

Comment:
`import` in NodeJS needs to specify the extension: "codesandbox/lib/api/define.js"